app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -153,8 +179,9 @@ public void onClick(View view) {
                     startActivity(GoToSubmitData);
                 } else {
                     // Check we have all the fields entered that are needed.  Otherwise, pop a TOAST message instead
-                    if (String.valueOf(edit_Match.getText()).isEmpty() || String.valueOf(edit_Team.getText()).isEmpty() || String.valueOf(edit_Name.getText()).isEmpty()
-                            || (spinner_StartPos.getSelectedItem().toString().equals(Globals.StartPositionList.getStartPositionDescription(Constants.DATA_ID_START_POS_DEFAULT)) && checkbox_DidPlay.isChecked())) {
+                    if (String.valueOf(edit_Match.getText()).isEmpty() || spinner_Team.getSelectedItemPosition() < 0 || String.valueOf(edit_Name.getText()).isEmpty()
```

Comment:
This may be valid, just wanted to check.  If they enter a Match number, but we happen to have no match data, the spinner will have no values in it.  What is getSelectedItemPosition() return?  -1?

what if the above happens, and they override and add in a team.  I assume that would return 0 (or 1?) and we're okay here?

Reply:
If we don't have match data, the spinner isn't empty it just has one option: "None"